addons/im_livechat/static/src/models/public_livechat_global.js

```diff
@@ -28,6 +29,12 @@ registerModel({
         },
     },
     recordMethods: {
+        async loadQWebTemplate() {
+            const templates = await this.messaging.rpc({ route: '/im_livechat/load_templates' });
+            for (let template of templates) {
```

Comment:
while you are at it:
```suggestion
            for (const template of templates) {
```

Reply:
change done